components/events/events-nav.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+var EventsNav = React.createClass({
```

Comment:
This file is the psudo nav, inside the hero unit, to switch between the two pages.

I was going to use mofo-ui but I needed routes for this too, so I'm just using the css for the moment.